src/utils/privacy/PIIDetector.ts

```
@@ -0,0 +1,255 @@
+        return value.replace(/\D/g, '').length === 9 ? 0.95 : 0.8;
+      case PIIType.CREDIT_CARD:
+        return this.isValidCreditCard(value) ? 0.9 : 0.6;
+      default:
+        return 0.8;
+    }
+  }
+
+  private maskValue(type: PIIType, value: string): string {
+    switch (type) {
+      case PIIType.EMAIL:
+        const [local, domain] = value.split('@');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)